packages/bsky/src/api/well-known.ts

```diff
@@ -4,31 +4,38 @@ import AppContext from '../context'
 export const createRouter = (ctx: AppContext): express.Router => {
   const router = express.Router()
 
-  router.get('/.well-known/did.json', (_req, res) => {
-    const hostname = ctx.cfg.publicUrl && new URL(ctx.cfg.publicUrl).hostname
-    if (!hostname || ctx.cfg.serverDid !== `did:web:${hostname}`) {
-      return res.sendStatus(404)
-    }
-    res.json({
-      '@context': ['https://www.w3.org/ns/did/v1'],
-      id: ctx.cfg.serverDid,
-      verificationMethod: [
-        {
-          id: `${ctx.cfg.serverDid}#atproto`,
-          type: 'Multikey',
-          controller: ctx.cfg.serverDid,
-          publicKeyMultibase: ctx.signingKey.did().replace('did:key:', ''),
-        },
-      ],
-      service: [
-        {
-          id: '#bsky_notif',
-          type: 'BskyNotificationService',
-          serviceEndpoint: `https://${hostname}`,
-        },
-      ],
+  const did = ctx.cfg.serverDid
+  if (did.startsWith('did:web:')) {
+    const hostname = did.slice('did:web:'.length)
+    const serviceEndpoint = `https://${hostname}`
+
+    router.get('/.well-known/did.json', (_req, res) => {
+      res.json({
+        '@context': ['https://www.w3.org/ns/did/v1'],
+        id: did,
+        verificationMethod: [
+          {
+            id: `${did}#atproto`,
+            type: 'Multikey',
+            controller: did,
+            publicKeyMultibase: ctx.signingKey.did().replace('did:key:', ''),
+          },
+        ],
+        service: [
+          {
+            id: '#bsky_notif',
+            type: 'BskyNotificationService',
+            serviceEndpoint,
+          },
+          {
+            id: '#bsky_app_view',
```

Comment:
My thinking on this was `#bsky_api` for the `id`, and `BskyAppView` for the `type`.

Maybe that is inconsistent though? I do like the terseness and generality of "API" instead of "AppView" in the `id`. If we stuck with AppView terminology, at least removing the second underline (so `#bsky_appview`) reads clearer to me.

For the "type", I feel like we have settled on "AppView" being a noun that is a type of service, so the extra suffix is unnecessary.